raiden-ts/src/services/epics.ts

```diff
@@ -695,3 +706,146 @@ export const monitorRequestEpic = (
       ),
     ),
   );
+
+export const udcWithdrawRequestEpic = (
+  action$: Observable<RaidenAction>,
+  state$: Observable<RaidenState>,
+  { userDepositContract, address, log, signer, config$, provider }: RaidenEpicDeps,
+): Observable<udcWithdrawPlanned | udcWithdraw.failure> => {
+  return action$.pipe(
+    filter(udcWithdraw.request.is),
+    withLatestFrom(from(userDepositContract.functions.balances(address))),
```

Comment:
This won't work. `withLatestFrom` would fire only once (at subscription time), not stay up to date, and if a request comes right away, this could suppress it because the result would still be undefined.
Instead, what you want os something like:
```suggestion
    mergeMap((action) => from(userDepositContract.functions.balances(address).then((balance) => [action, balance] as const))),
```

Reply:
OK that makes sense thank you